dusty/commands/test.py

```diff
@@ -143,10 +143,8 @@ def _run_tests_with_image(client, expanded_specs, app_or_lib_name, test_command,
 
     for service_container in previous_container_names:
         log_to_client('Stopping service container {}'.format(service_container))
-        try:
-            client.stop(service_container, timeout=1)
-        except Exception as e:
-            log_to_client('Exception stopping service container {}: {}'.format(service_container, str(e)))
+        client.kill(service_container)
```

Comment:
How would you persist something using a testing service container?   You would have to have compose spec set volumes to a mounted volume right?   This seems like a use case we do not want to encourage.  We want are trying to isolate these test runs right?